draft-ietf-webtrans-http3.md

```diff
@@ -323,6 +322,25 @@ However, when proxying the datagrams, the hop-by-hop MTUs can vary.
 TODO: Describe how the path MTU can be computed, specifically propagation across
 HTTP proxies.
 
+## Buffering Incoming Streams and Datagrams
+
+In WebTransport over HTTP/3, the client MAY send its SETTINGS frame, a
+WebTransport CONNECT request, a WebTransport data stream and a WebTransport
+datagram all within a single flight.  As those can arrive out of order, a
```

Comment:
This makes it sounds like the limit is 1 data stream and 1 datagram.  I think you want to say the client can send any number of any type.

Reply:
Good point. Fixed.